src/Compilers/CSharp/Test/Symbol/Symbols/InterfaceOverriddenOrHiddenMembersTests.cs

```diff
@@ -819,5 +819,361 @@ public interface Test { }
 }";
             CreateCompilationWithMscorlib(source).VerifyDiagnostics();
         }
+
+        [Fact]
+        [CompilerTrait(CompilerFeature.ReadonlyReferences)]
+        public void HidingMethodWithRefReadOnlyParameterWillProduceAWarning()
+        {
+            var code = @"
+interface A
+{
+    void M(ref readonly int x);
+}
+interface B : A
+{
+    void M(ref readonly int x);
+}";
+
+            var comp = CreateCompilationWithMscorlib(code).VerifyDiagnostics(
+                // (8,10): warning CS0108: 'B.M(ref readonly int)' hides inherited member 'A.M(ref readonly int)'. Use the new keyword if hiding was intended.
+                //     void M(ref readonly int x);
+                Diagnostic(ErrorCode.WRN_NewRequired, "M").WithArguments("B.M(ref readonly int)", "A.M(ref readonly int)").WithLocation(8, 10));
+
+            var aClass = comp.GlobalNamespace.GetMember<NamedTypeSymbol>("A");
+            var bClass = comp.GlobalNamespace.GetMember<NamedTypeSymbol>("B");
+
+            var aMethod = aClass.GetMember<MethodSymbol>("M");
+            var bMethod = bClass.GetMember<MethodSymbol>("M");
```

Comment:
Minor point: Can use `var aMethod = comp.GetMember<MethodSymbol>("A.M");` in each case.